queries/elixir/highlights.scm

```diff
@@ -181,38 +181,34 @@
   ])
 
 ; Module attributes
-(unary_operator 
-  operator: "@" @constant 
+(unary_operator
+  operator: "@" @constant
   operand: [
     (identifier) @constant
     (call target: (identifier) @constant)])
 
 ; Sigils
-(sigil 
+(sigil
   "~" @string.special
   ((sigil_name) @string.special) @_sigil_name
-  quoted_start: _ @string.special 
+  quoted_start: _ @string.special
   quoted_end: _ @string.special
   ((sigil_modifiers) @string.special)?
   (#not-any-of? @_sigil_name "s" "S"))
 
-(sigil 
+(sigil
   "~" @string
   ((sigil_name) @string) @_sigil_name
-  quoted_start: _ @string 
+  quoted_start: _ @string
   (quoted_content) @string
   quoted_end: _ @string
   ((sigil_modifiers) @string)?
   (#any-of? @_sigil_name "s" "S"))
 
 ; Documentation
-(unary_operator 
-  operator: "@" @comment 
-  operand: (call 
-    target: (((identifier) @comment) @_identifier)
-    (arguments [
-      (string) @comment
-      (charlist) @comment
-      (boolean) @comment
-    ]))
+(unary_operator
+  operator: "@" @comment
+  operand: (call
+    target: ((identifier) @_identifier)
+    _) @comment
```

Comment:
@jonatanklosko I have updated it. Unfortunately now it doesn't highlight strings as a comment:

<img width="303" alt="Zrzut ekranu 2021-11-2 o 15 31 05" src="https://user-images.githubusercontent.com/291639/139867288-2942d6db-ae70-44b6-8412-e749040fca13.png">

Reply:
This is what I get on master.

<img width="268" alt="Zrzut ekranu 2021-11-2 o 16 08 36" src="https://user-images.githubusercontent.com/291639/139874233-88f0b363-a2d6-40c3-bd0a-32b55bc89ee9.png">

And if I move that query above `(string) @string` then it is not applied at all.

Query like:

```scheme
; Documentation
(unary_operator 
  operator: "@" @comment 
  operand: (call 
    target: (((identifier) @comment) @_identifier)
    (arguments [
      (string) @comment
      (charlist) @comment
      (boolean) @comment
    ])
  (#any-of? @_identifier "moduledoc" "typedoc" "shortdoc" "doc")))
```

Nor

```scheme
; Documentation
(unary_operator 
  operator: "@" @comment 
  operand: (call 
    target: (((identifier) @comment) @_identifier)
    (arguments [
      (string) @comment
      (charlist) @comment
      (boolean) @comment
    ]
  (#any-of? @_identifier "moduledoc" "typedoc" "shortdoc" "doc"))))
```

Do not work either. It seems that there is no way to avoid matching the inner query fragments.

Reply:
Wait, so what's wrong master version then? That it ignores sigils or something else I miss?